util/zenfs.cc

```diff
@@ -53,27 +53,6 @@ bool zenfs_format_path(std::string &path, bool directory) {
   /* if the path is a directory, ensure that it ends with a slash */
   if (directory && path.back() != '/') path = path + "/";
 
-  /* relative paths are not supported */
```

Comment:
After this change the function does not do much and will never fail, so I think we cshould just drop this function and open-code the slash additions in the two places it is used..

Reply:
I disagree, removing a function to duplicate a line of code does not seem right to me.

Reply:
A one-liner add adding a slash if needed does not mandate a function i think, but it does not hurt. the function needs to be rewritten in any case.

Reply:
This function in its current form was returning always true and i called from list, backup, restore paths, but it is not necessary from "zenfs backup" path. So I have removed that.
So in place checks of "/" is needed in only 2 places (list, restore).

I have moved this check to a helper function.